frontend/src/pages/index.module.scss

```
@@ -79,21 +79,40 @@
   .steps {
     display: flex;
     flex-direction: column;
-    gap: $spacing-2xl;
+    gap: $layout-xl;
```

I think the idea behind using Protocol tokens is to keep things consistent, even if they were to change in the future. And though that ship has probably mostly sailed already, I think it's still good to try and not make a habit of using tokens in situations other than their intended use, if they happen to have the value (at this point in time) that you're looking for.

In other words, I'd avoid using a `$layout-` token for spacing. That means we can either say we're bound to the smaller spacing by Protocol, or (probably more constructively :slightly_smiling_face:) go for this compromise:

```suggestion
    gap: 2 * $spacing-2xl;
```

---

Hmm.. I think I'm still entirely unclear about the use cases for spacing v. layout. I remember a good rule of thumb was spacing for individual modules (margins/padding) and layout for spacing between elements/modules.

---

Oh! Actually, it might be that _I_ misinterpreted this token - I think I was confusing it with `$content-`. Then carry on :)

@maxxcrawford Could you confirm whether the interpretation of those tokens above is correct? Then I can use them correctly in the future as well :)